### PR TITLE
Updated interface images for backdrop, file type and menu buttons.

### PR DESCRIPTION
**Pull request by cristofercruz**
I've made a few changes for your consideration to polish up the interface a bit.

- New stylized backdrop image illustrated from a photo I took
- New menu bar buttons and highlight
- Tweaked file type images to work better against dark backgrounds
- Adjusted position of date/time and commit/revision

### CODE REVIEW

File: cube/swiss/source/images/buttons/btns.h [hunk 1]
```diff
@@ -17,8 +17,8 @@
 #define BTNEXIT_WIDTH   (100)
 #define BTNEXIT_HEIGHT  (38)
 
-#define BTNHILIGHT_WIDTH   (36)
-#define BTNHILIGHT_HEIGHT  (38)
+#define BTNHILIGHT_WIDTH   (67)
+#define BTNHILIGHT_HEIGHT  (49)
```
Comment (Extrems) [inline cube/swiss/source/images/buttons/btns.h]: Textures must be a multiple of 4x4.

Comment (cristofercruz) [reply cube/swiss/source/images/buttons/btns.h]: I've adjusted the highlight button texture to 4x4 multiple

**Comment by Extrems** [comment]
I'd merge this in a heartbeat, but I think I should wait for @emukidid's opinion.

**Comment by cristofercruz** [comment]
Sounds good, thanks for the feedback.

**Comment by cristofercruz** [comment]
> It's probably too late, but if you could provide the PSD files, particularly of the backdrop, that'd be great.

Not a problem, I’ll dig em up this weekend. Should I add them into the repo and do a PR?

**Comment by Extrems** [comment]
That'd be best, yeah.

**Comment by Extrems** [comment]
I managed to erase the rounded rectangles from the backdrop with fairly good precision so they can be composited at run-time instead, but I'd still appreciate the source file.

**Comment by cristofercruz** [comment]
> I managed to erase the rounded rectangles from the backdrop with fairly good precision so they can be composited at run-time instead, but I'd still appreciate the source file.

Raised PR #662 to include the PSD source. The source file needed some cleanup to get to a sharable state so I also updated the tif based on the source PSD.

**Pull request merged**